frontend/src/setupTests.ts

```diff
@@ -3,3 +3,7 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
This needs `yarn add -D @types/enzyme-adapter-react-16`

Reply:
indeed!